sdk/batch/Microsoft.Azure.Batch/tests/ProtocolTests/IntegrationJobApplicationPackageReferencesTests.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Honestly if it's easier you can just delete this entire ProtocolTests project. It's all live tests though so if you don't you should flag it as such.

If you don't delete it I will shortly.

Reply:
Ok